libs/cardano-protocol-tpraos/cardano-protocol-tpraos.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name:          cardano-protocol-tpraos
-version:       1.0.1.0
+version:       1.0.2.0
```

`1.0.2.0` is already on CHaP
```suggestion
version:       1.0.3.0
```

---

good catch, thank you!

---

I will fix this in the PR to master (#3410) as well